frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {
+  getContainerImageByDisk,
+  getURLSourceByDisk,
+  getPVCSourceByDisk,
+  getWindowsToolsURLByDisk,
+} from '../../selectors/vm/selectors';
+
+const getDiskSource = (vm, { name }) => {
```

Comment:
this is a component so it should be 
```suggestion
const DiskSource = ({vm, disk}) => {
```
also please add types (`React.FC`...)